README.md

```diff
@@ -1 +1,209 @@
-# React-Group
\ No newline at end of file
+<a name="readme-top"></a>
+
+- [📖 Space Travellers's Hub ](#-space-travellerss-hub-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [INSTALL](#install)
+    - [Usage](#usage)
+    - [Testing](#testing)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Space Travellers's Hub <a name="about-project"></a>
+
+**Space traveler's hub** is a single-page website where users can reserve rockets and join space missions.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+- React.js
+- Redux, Redux Toolkit
+- React-Bootstrap
+- Jest
+- Node.js
+- HTML/CSS/SCSS
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **You can reserve rocket**
+- **You can joing missions**
+- **You can view reservation on my profile page**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+ - [Live Demo Link]() 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- Hey, since you've not deployed the app, kindly remove the live demo link as its currently broken to make the readme more professional.